### PR TITLE
Feat/collection groups

### PR DESCRIPTION
**Pull request by mingard**

### CODE REVIEW

File: dadi/frontend/containers/App/App.jsx [hunk 1]
```diff
@@ -75,6 +75,10 @@ class App extends Component {
 
   render() {
     const {state, history} = this.props
```
Comment (eduardoboucas) [inline dadi/frontend/containers/App/App.jsx]: Missing line after initialisation.

File: dadi/frontend/containers/App/App.jsx [hunk 2]
```diff
@@ -93,28 +97,51 @@ class App extends Component {
...
+          </Main>
+        }
       </div>
     )
   }
 
+  groupRoutes() {
+    const {state} = this.props
+    const paths = []
+    state.app.config.apis[0].menu.forEach(item => {
```
Comment (eduardoboucas) [inline dadi/frontend/containers/App/App.jsx]: Missing line after initialisation.

File: dadi/frontend/containers/App/App.jsx [hunk 3]
```diff
@@ -93,28 +97,51 @@ class App extends Component {
...
+      if (typeof item === 'string') {
+        paths.push(':collection')
+      } else {
+        item.collections.map(collection => {
+          paths.push(`${slugify(item.title)}/:collection`)
+        })
+      }
+    })
+    return paths
```
Comment (eduardoboucas) [inline dadi/frontend/containers/App/App.jsx]: Missing line before `return`

**Comment by eduardoboucas** [comment]
Other than minor cosmetic things 💅 💄 , looks great!

File: dadi/frontend/containers/App/App.jsx (hunk 2 above)
Comment (mingard) [inline dadi/frontend/containers/App/App.jsx]: where should this line be?

Comment (eduardoboucas) [inline dadi/frontend/containers/App/App.jsx]: After line 132.

**Pull request merged**